packages/core/src/execution-engine/node-execution-context/utils/signature-helpers.ts

```diff
@@ -0,0 +1,34 @@
+import crypto from 'crypto';
+import type { Request } from 'express';
+
+import { WAITING_TOKEN_QUERY_PARAM } from '../../../constants';
+
+/**
+ * Generate signature token from url and secret
+ */
+export function generateSignatureToken(url: string, secret: string) {
```

Comment:
HMAC is not really a token. It's a signature. So this could be 
```suggestion
export function generateUrlSignature(url: string, secret: string) {
```